### PR TITLE
Fix handling of kwargs from common method invocations

### PR DESCRIPTION
**Pull request by nairbv**
When kwargs are specified in a test defined via common_method_invocations, it doesn't work if there isn't also a positional argument (`{'foo':'foo'}` without a positional arg generates a python call like: `self.method(, foo=foo)`, erroring on the `,`). I wanted to test something in a different PR and noticed I couldn't.

Also fixed some flake8 warnings I was seeing locally.

I replaced `lambda x: x` with `ident` since it seems a bit cleaner to me, but happy to revert that if others don't agree?

### CODE REVIEW

File: test/common_methods_invocations.py [hunk 1]
```diff
@@ -251,6 +253,7 @@ def method_tests():
         ('clamp', (), (0, 1), 'scalar', (True,)),
         ('clamp', (), (None, 0.5), 'min_scalar', (True,)),
         ('clamp', (), (0.5, None), 'max_scalar', (True,)),
+        ('clamp', (S, S), (), 'max_scalar_kwarg', (True,), (), (), ident, {'max': 1}),
```
Comment (nairbv) [inline test/common_methods_invocations.py]: This is an example test entry I added that we couldn't handle without the changes to test_jit.py and test_autograd.py (the rest are test changes below are just the ident replacement).

File: test/common_methods_invocations.py [hunk 2]
```diff
@@ -830,8 +833,6 @@ def method_tests():
         ('__getitem__', torch.randn(S, S, S), (dont_convert([[0, 2, 3], [1, 3, 3],
                                                              torch.LongTensor([0, 0, 2])]),), 'adv_index_var'),
     ]
-# TODO: clamp with min/max
-
```
Comment (nairbv) [inline test/common_methods_invocations.py]: checked the git log and it looks like when this comment was added there was only a single clamp test above.

**Review by gchanan — APPROVED** [review]
this looks fine to me -- @suo can you have someone look at the JIT side changes?

**Review by suo — APPROVED** [review]
lgtm

**Pull request closed without merging**